src/components/BeaconResultTileDetailsV2.vue

```diff
@@ -16,22 +16,27 @@
       </b-button>
       <div v-if="display" class="details-rows">
         <b-table
-          :data="results"
+
+          :data="parsedResults"
+
           :striped="true"
           ref="table"
           class="column details-table"
           detailed
+
+          :hover="true"
           @details-open="row => $buefy.toast.open(`Expanded ${row.id}`)"
         >
-          <b-table-column label="Id" field="id" v-slot="results">
-            <b-tag
-              class="accessibility-green-tag"
-              title="Dataset is in public access"
-              ><b>{{ getId(results) }}</b></b-tag
+          <b-table-column label="Id" field="id" v-slot="props">
+            <b-tag class="accessibility-green-tag"
+              ><b>{{ getId(props) }}</b></b-tag
             >
           </b-table-column>
-          <b-table-column label="Access">
-            <template v-slot:default="results">
+          <!-- remove tags if accessibility is not visible in baecon2 --->
```

Comment:
lines 35-64 look commented out ? - why is that
if we cannot use in beacon2 maybe we should remove it ?

Reply:
```suggestion
          <!-- remove tags if accessibility is not visible in beacon2 --->
```